### PR TITLE
added eslint & prettier

### PR DESCRIPTION
**Pull request by biw**
Added a basic eslint and prettier config. I tried to keep the style of the repo as much as possible, but if there are any changes you'd like to me make please let me know!

By adding eslint, I also found one bug that I didn't know how to proceed with. In the following code in `processors.js`, `frameWidth` is never defined:
https://github.com/xenova/transformers.js/blob/e628c0a957d4b40810e73ea50a786142327998a2/src/processors.js#L416-L418

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -332,14 +351,14 @@ function getTopItems(items, top_k = 0) {
     // if top == 0, return all
 
     items = Array.from(items)
-        .map((x, i) => [i, x])            // Get indices ([index, score])
-        .sort((a, b) => b[1] - a[1])      // Sort by log probabilities
+        .map((x, i) => [i, x]) // Get indices ([index, score])
+        .sort((a, b) => b[1] - a[1]); // Sort by log probabilities
```
Comment (kungfooman) [inline src/utils.js:355]: Is the linter able to leave this alone? Aligned comments make it more readable and easier to edit using multi-cursors.

File: src/utils.js [hunk 2]
```diff
@@ -366,26 +385,25 @@ function magnitude(arr) {
...
-        return Object.setPrototypeOf(closure, new.target.prototype)
+        let closure = function (...args) {
+            return closure._call(...args);
+        };
+        return Object.setPrototypeOf(closure, new.target.prototype);
     }
 
-    _call(...args) {
-        throw Error('Must implement _call method in subclass')
+    _call() {
```
Comment (kungfooman) [inline src/utils.js:396]: Feels strange to just delete arguments, for documentation and it also has practical effects:

image

**Comment by xenova** [comment]
Thanks for the PR! The repo has changed quite a bit since you made the PR, so, a lot of the files are quite outdated. However, it is still important to get a linter and prettier included as it will help keep the repo organized and structured.

If you'd like, you could maybe try update your branch to have all the latest changes, then we can review the linter and prettier options.

**Comment by xenova** [comment]
> I will take a look at this later tonight and clean things up 😄

I'm still keen to add this, so, if you're able to fix the conflicts, I'll be able to merge it more easily 👍

**Pull request closed without merging**